src/bbs2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -65,6 +68,16 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
         description: "The SSH port (default: 22).",
         getDefaultValue: () => 22);
 
+    public Option<string> SmbUser { get; } = new(
+        name: "--smb-user",
+        description: "The SMB user to be used for downloading the export archive off of the Bitbucket server." +
```

Comment:
I think this makes it a bit clearer what this does:

```suggestion
        description: "The SMB user used for authentication when downloading the export archive from the Bitbucket Server instance." +
```

Reply:
addressed in 00ba81e9e3a38610372fbdb52f77fea5d1b21662